scripts/gas-refund-program/staking/stakes-tracker.ts

```diff
@@ -51,46 +51,18 @@ export default class StakesTracker {
     );
     assert(startTimeSM < endTime, 'startTimeSM should be less than endTime');
 
-    const [startBlockSPSP, startBlockSM, endBlock] = await Promise.all([
-      blockInfo.getBlockAfterTimeStamp(startTimeSPSP),
-      blockInfo.getBlockAfterTimeStamp(startTimeSM),
-      blockInfo.getBlockAfterTimeStamp(endTime),
```

Comment:
Little side effect: endTime -> endBlock is resolved twice. In context of gas refund this represents one extra req at script initialisation.
This can be addressed with caching later.